### PR TITLE
must-gather: Readme update with how to contribute

### PR DESCRIPTION
**Pull request by bindrad**
Signed-off-by: Dhruv Bindra <dbindra@redhat.com>

### CODE REVIEW

File: must-gather/README.md [hunk 1]
```diff
@@ -41,4 +41,37 @@ You will get a dump of:
...
+3. Make changes.
+4. Ensure your commit messages include sign-off.
+5. Push your changes to a branch in your fork of the repository.
+6. Submit a pull request to the `openshift/ocs-operator` repository.
+7. Work with the community to make any necessary changes through the code
+   review process (effectively repeating steps 3-7 as needed).
+
+#### Commit and Pull Request Messages
+
+- The message must begin with must-gather and a colon followed by single line description.
```
Comment (rajatsing) [inline must-gather/README.md:62]: add a subsection and give an example such as.
```
must-gather: adding processes to the background
```
The more example you give the better it is for new desvs who want to contribute, will understand.

File: must-gather/README.md [hunk 2]
```diff
@@ -41,4 +41,37 @@ You will get a dump of:
...
+    the summary.
+  - It must describe why the change is necessary or useful.
+  - It must describe how the change was implemented.
+  - It must reference any open Issues the change addresses.
+  - It must wrap at 80 characters.
+  - *For very small changes it may be acceptable to omit the longer description.
+    Please remember, that it is easy for any developer to believe their code is
+    "self-documenting" when it is not. Adding a description will help future
+    maintainers of the project.*
+- The message must end with a sign-off
```
Comment (rajatsing) [inline must-gather/README.md:74]: when you finish writing all the points, add an example showing how a proper commit message looks like.
Give an example of how to signoff a PR
` git commit --signoff` and then how to write the full commit message finally.

File: must-gather/README.md [hunk 3]
```diff
@@ -41,4 +41,64 @@ You will get a dump of:
...
+```
+<subsystem>: <what changed>
+<BLANK LINE>
+<why this change was made>
+<BLANK LINE>
+<footer>
+```
+
+Pull request messages should follow the same format, but for the entire set of
+changes contained in the pull request.
```
Comment (rajatsing) [inline must-gather/README.md:104]: ok, so one small nit, move the format above before giving the example.
Like the devs would want to look at the commit structure first then they can see the actual example,right?.
You want to look at the concept and then look at some examples of how to do it. It is like that way.

**Comment by rajatsing** [comment]
everything else looks good, apart from the swapping structure

File: must-gather/README.md (hunk 1 above)
Comment (crombus) [reply must-gather/README.md:62]: also recommend small case for `must-gather` tag.

**Review by crombus — CHANGES_REQUESTED** [review]
Thanks for the PR. A good starting point. Few changes required.

for ocs-operator we have https://github.com/openshift/ocs-operator/blob/master/CONTRIBUTING.md.  see if you can use this.

File: must-gather/README.md [hunk 4]
```diff
@@ -41,4 +41,64 @@ You will get a dump of:
...
+    <BLANK LINE>
+    <why this change was made>
+    <BLANK LINE>
+    <footer>
+    ```
+  - The comment must begin with must-gather and a colon followed by single line of description.
+    The example message:
+    ```
+    must-gather: Readme update with how to contribute  
+
```
Comment (crombus) [inline must-gather/README.md:96]: repetitive line .

**Review by crombus — CHANGES_REQUESTED** [review]

**Review by crombus — CHANGES_REQUESTED** [review]
@bindrad change of plan. I feel we will follow the same pattern as ocs-operator does so just mention the ocs-operator and say that follow the same standards for raising a PR. Only thing we need to add is the tag `must-gather:`.

**Review by crombus — APPROVED** [review]
/lgtm

**Pull request merged**